rust-format-diff.sh

```diff
@@ -0,0 +1,8 @@
+for d in $(find . -name "Cargo.toml" -printf '%h\n')
+do
+    (cd $d; cargo fmt -- --write-mode=diff)
+    status=$?
+    if [ $status -ne 0 ] && [ $status -ne 101  ]; then # 101 is due to a bug in rustfmt that the ./vm package is triggering
+        exit $status;
+    fi
+done
```

Comment:
I looked for an option like that and wasn't able to find it.  I'll give it a try when I get a chance.